docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "2"
+services:
+    app:
+        container_name: restfulness
+        build: ./
+        links:
+            - db
+        ports:
+            - "5000:5000"
+
+    db:
+        container_name: mysql
+        image: mysql:8.0
+        ports:
+            - "32000:3306" # I have mysql-server installed on my machine, so using other port instead of 3306
+        environment:
+            MYSQL_ROOT_PASSWORD: root
```

Comment:
When I used `MYSQL_ALLOW_EMPTY_PASSWORD`, Docker showed a warning that it's not good to leave MySql without password, also, in documentation that you linked, it says:

> NOTE: Setting this variable to yes is not recommended unless you really know what you are doing, since this will leave your MySQL instance completely unprotected, allowing anyone to gain complete superuser access.

Also for `MYSQL_ROOT_PASSWORD` it says:

> This variable is **mandatory**

So, I decided to use `MYSQL_ROOT_PASSWORD` and `MYSQL_ONETIME_PASSWORD` both together (So when initialization completed, root's password will be expired) and then, using `MYSQL_USER` and `MYSQL_PASSWORD`, I created a user so that app can use that user to access to _only_ `restfulness` database.

Also I updated `README`, so if anyone decided to change this setting, can do it easily.